src/feature/dirauth/dirauth_config.c

```diff
@@ -443,6 +437,12 @@ dirauth_options_validate(const void *arg, char **msg)
   t = format_recommended_version_list(options->RecommendedServerVersions, 1);
   tor_free(t);
 
+  if (options->TestingAuthDirTimeToLearnReachability < 0) {
```

Comment:
This code is only run by the unit tests, see my earlier comments about interval.